product-web-app/views/job-specification.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JobSpec</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css">
```

Comment:
use the same version of the bootstrap - see other files

Reply:
i use the same bootsrap as Jakub I think

Reply:
yup it's 4.3.1 all over the place (there seems to be a different version on one of the branches, but not here)